### DISCUSSION TITLE
How to get the last & current iteration items

### DISCUSSION THREAD

**Discussion by exaucae** [gr2m/github-project#149] [Q&A] [answered]
I want to move the last iteration items of my project into the current iteration. Is there a way I can get the items that belong to those iterations?

**Comment by gr2m** [comment]
Iteration fields are the last one that is not supported yet, but we have an open pull request: https://github.com/gr2m/github-project/pull/95. Any help to finish it up would be much appreciated

**Reply by exaucae** [reply]
is there anything left to do in the PR? Looks like we've reached a working state: https://github.com/gr2m/github-project/issues/94#issuecomment-1445479967

**Reply by gr2m** [reply]
Yes, I since commented on the pull request, the author didn't get back to me. Would you like to try to take it over the finish line?
https://github.com/gr2m/github-project/pull/95#issuecomment-1783962104

**Reply by blombard** [reply]
Sorry for letting this fall into limbo. I fixed the failing tests :)

**Reply by gr2m** [reply, marked as answer]
Iteration fields are supported as of [v5.4.0](https://github.com/gr2m/github-project/releases/tag/v5.4.0). 

@exaucae can you confirm it's working for you?
